app/helpers/dashboard_catalog_helper.rb

```diff
@@ -1,123 +1,44 @@
 # frozen_string_literal: true
 
+# TODO: this will be going away in favor of Dashboard::CatalogService and ViewComponents
+
 # helper methods for dashboard pertaining to catalog functionality
 module DashboardCatalogHelper
+  # used by catalog_status partials
   def catalog_ok?
     num_preserved_objects == num_complete_moabs &&
       num_object_versions_per_preserved_object == num_object_versions_per_complete_moab
   end
 
-  def storage_root_info # rubocop:disable Metrics/AbcSize
```

Comment:
anything dropped here is in the new service class and is only used by the new ViewComponents